kubernetes/coredns.yaml.sed

```diff
@@ -96,6 +96,8 @@ spec:
           effect: NoSchedule
         - key: "CriticalAddonsOnly"
           operator: "Exists"
+      nodeSelector:
+        beta.kubernetes.io/os: linux
```

Comment:
@MarcPow These changes have already been added.
https://github.com/coredns/deployment/blob/1b4fe411a601a090722aec9503167ee8cc8c706f/kubernetes/coredns.yaml.sed#L99-L100

Reply:
My apologies Chris/Sandeep.   I forked the repository a couple of days ago, intending to do this then, but when I applied it today, I didn't update my fork - so I thought they were still un-applied.

Silly mistake.  I'll yank this.